server/go/model_api_item.go

```diff
@@ -19,6 +19,8 @@ type ApiItem struct {
 	Service ApiItemService `json:"service"`
 
 	Version string `json:"version"`
+
+	Crunch42URL string `json:"crunc42url"`
```

Comment:
Mis-spelling: Shouldn't it be `crunch42url` or rather `42crunch_url`?

Reply:
fixed it